base/observer_list_threadsafe.h

```diff
@@ -90,7 +90,11 @@ class BASE_EXPORT ObserverListThreadSafeBase
     }
   };
 
+#if defined(STARBOARD)
+  static const NotificationDataBase* GetCurrentNotification();
```

Comment:
We'd have to change callsites anyways if we're changing the signature, so I figured just not using the function was better

Reply:
Ahh. Now I see that you're not calling it anymore. Did a quick test, and this change isn't needed for the test to pass. I'll remove this from the PR.

Reply:
Not much left here.